### ISSUE TITLE
Generated Decorator Should be @PrimaryGeneratedColumn() - Maybe?

### ISSUE THREAD

**Issue by larkintuckerllc** [Kononnable/typeorm-model-generator#96]
Using TypeORM with following decorator (per their documentation)

```
  @PrimaryGeneratedColumn()
  public id: number;
```

behaves differently than the decorator generated from typeorm-model-generator

```
    @Column("integer",{ 
        generated:true,
        nullable:false,
        primary:true,
        name:"id"
        })
    id:number;
```

The difference appears to be that the former will return the generated id when saving a new entity where-as the second does not.

It is not clear if this is a bug in typeorm-model-generator (for not using the idiomatic decorator) or a bug in TypeORM for not handling the decorator you generate.

**Comment by Kononnable** [comment]
I think this is an error in typeorm. I think these notation should be equal.
```ts
@Column("int",{
  primary:true,
  generated:true
  })

@PrimaryGeneratedColumn()
```
Please make an issue about this in typeorm repo. @pleerock will tell us if for some reason behavior should vary.

However this issue looks like critical one, so I've hanged model generation to use @PrimaryGeneratedColumn decorator.

**Comment by Kononnable** [comment]
Implemented in `typeorm-model-generator@0.2.21`

**Comment by Kononnable** [comment]
Issue about the problem: typeorm/typeorm#2364
